sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2024-2025.bpmn

```diff
@@ -828,29 +822,63 @@
+      <bpmn:incoming>Flow_0n8rikf</bpmn:incoming>
+      <bpmn:outgoing>Flow_1gf8xyd</bpmn:outgoing>
+    </bpmn:parallelGateway>
+    <bpmn:sequenceFlow id="Flow_037lfnr" sourceRef="Gateway_0wuxqll" targetRef="Event_1mkiqso" />
+    <bpmn:sequenceFlow id="Flow_1vb0nqd" sourceRef="Gateway_0wuxqll" targetRef="Event_0asamuq" />
+    <bpmn:sequenceFlow id="Flow_0n8rikf" sourceRef="Event_0asamuq" targetRef="Gateway_08rr1xe" />
+    <bpmn:sequenceFlow id="Flow_1tkgnk6" sourceRef="Activity_1vwijyc" targetRef="Gateway_0wuxqll" />
+    <bpmn:parallelGateway id="Gateway_0wuxqll">
+      <bpmn:incoming>Flow_1tkgnk6</bpmn:incoming>
+      <bpmn:outgoing>Flow_037lfnr</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1vb0nqd</bpmn:outgoing>
+    </bpmn:parallelGateway>
+    <bpmn:scriptTask id="Activity_1vwijyc" name="Define Dependents">
+      <bpmn:extensionElements>
+        <zeebe:script expression="=// Consider only dependents who were born on or before the study end date.&#10;inputDataDependents[date(item.dateOfBirth) &#60;= date(offeringStudyEndDate)]" resultVariable="calculatedDataDependants" />
+        <zeebe:ioMapping>
+          <zeebe:input source="=//TODO: Student appeal variable needs to be considered here.&#10;if studentDataDependants = null &#10;then []&#10;else studentDataDependants" target="inputDataDependents" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_15xhu93</bpmn:incoming>
+      <bpmn:outgoing>Flow_1tkgnk6</bpmn:outgoing>
+    </bpmn:scriptTask>
+    <bpmn:intermediateThrowEvent id="Event_0asamuq" name="Calculate Financial Information">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if (is defined(appealsStudentFinancialInformationAppealData) and appealsStudentFinancialInformationAppealData != null)&#10;  then&#10;  appealsStudentFinancialInformationAppealData.taxReturnIncome &#10;else&#10;  studentDataTaxReturnIncome" target="calculatedDataTaxReturnIncome" />
+          <zeebe:output source="=if (is defined(appealsStudentFinancialInformationAppealData) and appealsStudentFinancialInformationAppealData != null)&#10;  then appealsStudentFinancialInformationAppealData.daycareCosts12YearsOrOver &#10;else&#10;  studentDataDaycareCosts12YearsOrOver" target="calculatedDataDaycareCosts12YearsOrOver" />
+          <zeebe:output source="=if (is defined(appealsStudentFinancialInformationAppealData) and appealsStudentFinancialInformationAppealData != null)&#10;  then appealsStudentFinancialInformationAppealData.daycareCosts11YearsOrUnder &#10;else&#10;  studentDataDaycareCosts11YearsOrUnder" target="calculatedDataDaycareCosts11YearsOrUnder" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1vb0nqd</bpmn:incoming>
+      <bpmn:outgoing>Flow_0n8rikf</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:intermediateThrowEvent id="Event_1mkiqso" name="Define PD/PPD Status">
```

Comment:
Converting this to a `Script task` and using a similar approach from `inputDataDependents` would make this logic simpler.
The reason for this request is because this PR intention is also to set the foundation for upcoming tasks and to be used as a reference, hence having those changes in place would be useful.

Reply:
Replaced the intermediate throw event with script task and it's simpler.